python/paddle/fluid/tests/unittests/test_top_k_v2_op.py

```diff
@@ -137,26 +141,6 @@ def setUp(self):
         self.outputs = {'Out': output, 'Indices': indices}
 
 
-class TestTopkOp6(OpTest):
-    def init_args(self):
-        self.k = 100
-        self.axis = 1
-        self.largest = True
-
-    def setUp(self):
-        self.op_type = "top_k_v2"
-        self.python_api = paddle.topk
-        self.dtype = np.float64
-        self.input_data = np.random.rand(80, 16384)
-        self.init_args()
-        self.inputs = {'X': self.input_data}
-        self.attrs = {'k': self.k, 'axis': self.axis, 'largest': self.largest}
-        output, indices = numpy_topk(
-            self.input_data, axis=self.axis, k=self.k, largest=self.largest
-        )
-        self.outputs = {'Out': output, 'Indices': indices}
-
-
```

Comment:
why delete this test

Reply:
Can not pass CI-Coverage because it will run out of time, and original test case did not have checkout output and grad so it can pass.